lib/std/special/compiler_rt/divtf3_test.zig

```diff
@@ -35,7 +35,7 @@ test "divtf3" {
     // NaN / any = NaN
     try test__divtf3(math.nan_f128, 0x1.23456789abcdefp+5, 0x7fff800000000000, 0);
     // inf / any = inf
```

Comment:
nit: this comment is wrong:  inf/0 and inf/NaN are not inf, but NaN and could be tested.

Reply:
I'm pretty sure +inf/0 **is** +inf? Seems to be. I can add one for NaN, which indeed results in NaN.